First/FightSimulation/EloFightSimulator.cs

```diff
@@ -63,6 +62,10 @@ private static double eloDelta(double eloW, double eloL)
             return delta;
         }
 
+        private static double getNetworkViewers(double elo1, double elo2)
+        {
```

Comment:
Id consider exponential distributions as well

Reply:
That's something Greg does